### DISCUSSION TITLE
WAU is not updating several MS Store Apps

### DISCUSSION THREAD

**Discussion by JasonCioffi** [Romanitho/Winget-AutoUpdate#624] [Q&A]
WAU just ran at logon and updated Windows Terminal and App Installer (both MS Store Apps), but ignored updating Photos, XboxIdentityProvider, WindowsStore, Calculator, Paint, StickyNotes, and Notepad.  All show having updates available in the Microsoft Store.  Is this normal?  Do I need to configure something special for WAU to check and apply updates for all MS Store apps?

Thanks in advance for the replies.

**Comment by KnifMelti** [comment]
This project only handles updates from the WinGet community repository.
Manifests:
https://github.com/microsoft/winget-pkgs/tree/master/manifests

App Installer gets upgraded from https://github.com/microsoft/winget-cli/releases and Windows Terminal from the community repository
